### PR TITLE
main: Add '--version' CLI flag

### PR DESCRIPTION
**Pull request by har7an**
Prints the current version as defined in the `Cargo.toml`.

### CODE REVIEW

File: espmonitor/src/main.rs [hunk 1]
```diff
@@ -43,6 +43,9 @@ fn parse_args() -> Result<Option<AppArgs>, Box<dyn Error>> {
     if args.contains("-h") || args.contains("--help") {
         print_usage();
         Ok(None)
+    } else if args.contains("-v") || args.contains("--version") {
```
Comment (kelnos) [inline espmonitor/src/main.rs:46]: GNU standard is generally a _capital_ `V` for the short option.  (Lowercase `v` is used for `--verbose`.)

**Review by kelnos — CHANGES_REQUESTED** [review]

**Comment by kelnos** [comment]
Thanks for this!  Probably should just move to a more full-featured command line parser like `clap` that does all this for us, but this is great for now.  Just one small change and I'm happy to merge.

**Comment by kelnos** [comment]
Oh -- it looks like the cargo integration doesn't do this by itself, so if you'd be interested in adding it there as well, that'd be great.

**Comment by har7an** [comment]
Thanks for the feedback. I added the requested change and copied the same code to the cargo-esptools. I also updated the usage message to reflect the new `--version` option.

**Comment by har7an** [comment]
> Probably should just move to a more full-featured command line parser like `clap` that does all this for us

I've got some experience with clap. Shall I go ahead and try to add this?

**Comment by kelnos** [comment]
Ok, we're working on an impl with `clap` now, so I guess we can close this out.

**Pull request closed without merging**